### ISSUE TITLE
Validation warning with `compress="xz"`, backup successful

### ISSUE THREAD

**Issue by petecooper** [sebastianfeldmann/phpbu#226]
I am trying `xz` compression instead of `gzip`, like this:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<phpbu xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://schema.phpbu.de/6.0/phpbu.xsd">
    <logging>
        <log type="json" target="/var/www/vhosts/example.com/www/backups/www.example.com-phpbu-log-hourly.json"/>
    </logging>
    <backups>
        <backup name="example.com database gzip hourly>">
                <source type="mysqldump">
                    <option name="databases" value="dbname" />
                    <option name="host" value="127.0.0.1" />
                    <option name="password" value="dbpass"/>
                    <option name="user" value="dbuser" />
                </source>
                <target dirname="/mnt/backups-01/backups/phpbu/example.com/www/hourly/database/" filename="www.example.com_hourly_%Y%m%d%H%i%s" compress="xz" />
                    <cleanup type="quantity">
                        <option name="amount" value="72" />
                    </cleanup>
                </backup>
        </backups>
</phpbu>
```

I use PHP 7.3.13 to run PHPBU, and the backup completes, but with a validation warning:

```
phpbu 6.0.3 by Sebastian Feldmann and contributors.

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 14:
  - Element 'target', attribute 'compress': [facet 'enumeration'] The value 'xz' is not an element of the set {'bzip2', 'gzip', 'zip', '7zip'}.
  - Element 'target', attribute 'compress': 'xz' is not a valid value of the local atomic type.

Time: 41 seconds, Memory: 4.00MB

OK (1 backup, 0 checks, 0 crypts, 0 syncs, 1 cleanup)
```

`xz` is installed, and accessible:

``` bash
$ xz -V
xz (XZ Utils) 5.2.2
liblzma 5.2.2
```

I would be grateful if you could advise what I need to do to fix the XML in order for it to validate correctly.

Thank you for this excellent software, and best wishes to you.

**Comment by sebastianfeldmann** [comment]
It is more what I have to do to fix the missing `xz` in the xsd :/
I'll update the `xsd` and release a new version.

**Comment by sebastianfeldmann** [comment]
I've updated the `xsd` file and added `xz` to the list of valid compression values.

Please update to `6.0.4` to validate against the latest `xsd` file. this should fix your issue.

**Comment by petecooper** [comment]
Issue fixed with `6.0.4` - thank you very much!

**Comment by sebastianfeldmann** [comment]
Thanks for the feedback ;)
